debug/playground/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "lwc-playground",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "dependencies": {
+    "eslint": "^9.15.0",
```

To get this working on a fresh install, I needed to add this:
```suggestion
  "dependencies": {
    "lightweight-charts": "file:../..",
    "eslint": "^9.15.0",
```